SamplesDashboard/SamplesDashboard/ClientApp/src/views/Repositories.tsx

```diff
@@ -324,16 +319,24 @@ function compare(a: any, b: any, isSortedDescending?: boolean) {
 function checkStatus(status: number) {
```

Comment:
This function can be reused. Consider making it into a functional component so that the descriptions and icons can be managed from a single source

Reply:
Yes, this applies to multiple functions. I will make them functional components for sure. I'll raise this as an issue too as it is a separate task